src/components/Calculator.js

```diff
@@ -1,43 +1,78 @@
+          <button type="button" className="button-section" value="+/-" onClick={() => { this.calculator(digitObj, '+/-'); }}>+/-</button>
+          <button type="button" className="button-section" value="%" onClick={() => { this.calculator(digitObj, '%'); }}>%</button>
+          <button type="button" className="button-section operator" value="÷" onClick={() => { this.calculator(digitObj, '÷'); }}>÷</button>
+          <button type="button" className="button-section" value="7" onClick={() => { this.calculator(digitObj, '7'); }}>7</button>
+          <button type="button" className="button-section" value="8" onClick={() => { this.calculator(digitObj, '8'); }}>8</button>
+          <button type="button" className="button-section" value="9" onClick={() => { this.calculator(digitObj, '9'); }}>9</button>
+          <button type="button" className="button-section operator" value="x" onClick={() => { this.calculator(digitObj, 'x'); }}>x</button>
+          <button type="button" className="button-section" value="4" onClick={() => { this.calculator(digitObj, '4'); }}>4</button>
+          <button type="button" className="button-section" value="5" onClick={() => { this.calculator(digitObj, '5'); }}>5</button>
+          <button type="button" className="button-section" value="6" onClick={() => { this.calculator(digitObj, '6'); }}>6</button>
+          <button type="button" className="button-section operator" value="-" onClick={() => { this.calculator(digitObj, '-'); }}>-</button>
+          <button type="button" className="button-section" value="1" onClick={() => { this.calculator(digitObj, '1'); }}>1</button>
+          <button type="button" className="button-section" value="2" onClick={() => { this.calculator(digitObj, '2'); }}>2</button>
+          <button type="button" className="button-section" value="3" onClick={() => { this.calculator(digitObj, '3'); }}>3</button>
+          <button type="button" className="button-section operator" value="+" onClick={() => { this.calculator(digitObj, '+'); }}>+</button>
         </div>
         <span className="end">
-          <button type="button" className="button-section">0</button>
-          <button type="button" className="button-section">.</button>
-          <button type="button" className="button-section operator">=</button>
+          <button type="button" className="button-section" value="0" onClick={() => { this.calculator(digitObj, '0'); }}>0</button>
+          <button type="button" className="button-section" value="." onClick={() => { this.calculator(digitObj, '.'); }}>.</button>
+          <button type="button" className="button-section operator" value="=" onClick={() => { this.calculator(digitObj, '='); }}>=</button>
         </span>
       </section>
     );
   }
 }
 
+Calculator.propTypes = {
+  updateState: PropTypes.func.isRequired,
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Linting is important to reduce errors and improve the overall quality of your code. Using lint tools can help you accelerate development and reduce costs by finding errors earlier. so please do not disable them since they are meant to help you :wink: